backend/protocol_rpc/app_lifespan.py

```
@@ -311,6 +316,20 @@ def get_session() -> Session:
         )
         await redis_subscriber.connect()
         await redis_subscriber.start()
+
+        # Register handler for validator change events
+        async def handle_validator_change(event_data):
```

Ruff (0.14.1)

Unused function argument: `event_data`

(ARG001)